includes/models/class-llms-rest-webhook.php

```diff
@@ -472,11 +499,10 @@ protected function schedule( $args ) {
 		 * @param array             $args  Numeric array of arguments from the originating hook.
 		 * @param LLMS_REST_Webhook $this  Webhook object.
 		 */
-		$delay = apply_filters( 'llms_rest_webhook_repeat_delay', 300, $args, $this );
+		$delay = apply_filters( 'llms_rest_webhook_repeat_delay', 3, $args, $this );
```

Comment:
Is this an intentional change or was it accidentally committed after debugging?

Reply:
shite

Reply:
fixed in  3775efa